docs/chapters/flow/index.md

```diff
@@ -224,7 +224,7 @@ def input_fn():
 input_fn = (b's' for _ in range(10))
 ```
 
-> Please note that the current Flow API does not support using `index()` and `search()` together in the same `with` scope. This is because the workflow of `index()` and `search()` are usually different and you cannot use one workflow for both tasks.
+> Please note that the current Flow API does not support using `index()` and `search()` together in the same `with` scope. This is because the workflow of `index()` and `search()` are usually different, and you cannot use one workflow for both tasks.
```

Comment:
If itis about proper closure of Peas/Pods, it makes sense. If it is about multiple clients under the same scope, that is allowed.

![image](https://user-images.githubusercontent.com/9050737/105344130-8f622600-5c08-11eb-99f9-4917aee842e3.png)

Reply:
Interesting. However, the only thing i changed was a comma `,` lol